src/main.rs

```diff
@@ -60,7 +61,11 @@ fn main() {
 
     let matches = match opts.parse(&args[1..]) {
         Ok(m) => m,
-        Err(f) => panic!(f.to_string()),
+        Err(e) => {
+          writeln!(&mut io::stderr(), "{}", e)
+              .expect("failed printing to stderr");
+          std::process::exit(1);
```

Comment:
hmm....using `error!` doesn't show any error information to the user :-(
So should we use `println!` followed by a `error!` statement?